oeps/oep-0001.rst

```diff
@@ -83,28 +85,23 @@ sure the idea is applicable to the entire community and not just the author.
 Just because an idea sounds good to the author does not mean it will work for
 most Open edX installations.
 
+Initial Submission
+------------------
+
 Once the Author has asked the Open edX community whether an idea has any chance
 of acceptance, a draft OEP should be submitted as a pull request against the
 `central OEP repository`_. To identify the draft proposal, the Author should
-check the numbered list of previous OEP pull requests and select the next
-available number. The pull request title must start with "OEP-XXX", which
+check the numbered list of previous OEP pull requests and select the next 
+available number. The pull request title must start with "OEP-XXX", which 
 claims that OEP number for the included proposal.
 
 .. _central OEP repository: http://github.com/edx/open-edx-proposals
 
-As updates are necessary, the OEP Author can update the pull request.
-
-OEPs can be reference pull requests to other Open edX repositories which
-will act as reference implementations.
-
-OEP Review & Resolution
------------------------
-
-After the Author drafts an OEP, and submits it as a pull request, they request
-an Arbiter from the edX Chief Architect. This Arbiter will be recorded in the
-"Arbiter" header on the OEP. The rest of Open edX community will be given the
-opportunity to comment on the OEP, with the Arbiter serving to keep the
-discussion on track and to evaluate when it has reached a final conclusion.
+After the Author drafts an OEP in a format in which they are comfortable, they
+request an Arbiter from the edX Chief Architect. This Arbiter will be recorded
```

Comment:
Does this also applies to product-related OEPs? Or would it be someone from product selecting the Arbiter in these cases?